build-info-extractor-go/src/main/java/org/jfrog/build/extractor/go/extractor/PathUtils.java

```diff
@@ -0,0 +1,78 @@
+package org.jfrog.build.extractor.go.extractor;
+
+import java.io.File;
+import java.util.regex.Pattern;
+
+/**
+ * @author BarakH
+ */
+public class PathUtils {
+
+    private static final Pattern PATTERN_SLASHES = Pattern.compile("/+");
+
+    /**
+     * @param path A file path
+     * @return Parent path of the input path as if it was a file. Empty string if the path has no parent.
+     */
+    public static String getParent(String path) {
+        if (path == null) {
+            return null;
+        }
+        File dummy = new File(path);
+        return formatPath(dummy.getParent());
+    }
+
+    /**
+     * Replaces all backslashes in the path to slashes.
+     *
+     * @param path A path to format
+     * @return The input path with all backslashes replaced with slashes. Return empty string if the input path is
+     * null.
+     */
+    public static String formatPath(String path) {
+        if (hasText(path)) {
```

Comment:
This if seems redundant. What would be the problem if path does not have text?

Reply:
removed